pipeline/train/configs/opustrainer/student.yml

```diff
@@ -8,21 +8,21 @@ train:
   - original 1.0
   - until original inf # General training until marian early stops
 
-
 modifiers:
 - UpperCase: 0.07 # Apply randomly to 7% of sentences
 - TitleCase: 0.05
-# do not use typos modifier for the student, it causes issues with alignments
-#- Typos: 0.05
-# TODO: enable noise for student when we switch to space tokenized alignments for Tags
-#- Noise: 0.0005
-#  min_word_length: 2 # Minimum word length for each word in the noisy sentence
-#  max_word_length: 5 # Maximum word length for each word in the noisy sentence
-#  max_words: 6 # Maximum number of words in each noisy sentence
-# TODO: enable inline noise, currently doesn't work because it requires space tokenized alignments
-# TODO: https://github.com/mozilla/firefox-translations-training/issues/261
-#- Tags: 0.05
-#  augment: 1
+- Typos: 0.05
+# inserts new noise sentences
+- Noise: 0.0005
```

Comment:
I see that you have docs in a follow-up PR, maybe you can refer to those? I would apply the same level of reasoning that I did on the training config yml, a light amount of explanation with a reference to a file with more information. You can apply this feedback in the following PR if you want.

Reply:
I would prefer consolidating the docs in markdown rather than spreading them through the code. I can add some short notes about the values though, but while we continue tuning them I don't think we'll be able to document a decision behind every hyperparameter we have.

Reply:
I'm happy to consolidate on markdown documentation, but would like to make sure we cross link it in the code where it's applicable so it's easy to jump to the docs.